src/store/configureStore.js

```diff
@@ -1,6 +1,6 @@
-import { configureStore } from '@reduxjs/toolkit';
-import reducer from './reducer';
+// import { configureStore } from '@reduxjs/toolkit';
+// import reducer from './reducer';
 
-export default configureStore({
-  reducer,
-});
+// export default configureStore({
+//   reducer,
+// });
```

Comment:
Modifying this code would break other team member's code, kindly stick to previously agreed code structure.